docs/operator.md

```diff
@@ -39,6 +39,7 @@ Notes:
 described in [cluster setup instructions using `cmk cluster-init`][cluster-init-op-manual].
 - The [cluster setup instructions using manually created Pods][indvidual-pods-op-manual] should only be used if and 
 only if running `cmk cluster-init` fails for some reason.
+- ___(multiple socket platforms)___ Dataplane pool shouldn't be spread within numa nodes, so `--num-dp-cores` value should be lower than number of cores on single socket.
```

Comment:
Agree.... i think that pool can spread across numa nodes, the question is about i.e. NIC presence on specific socket. Another case is how to spread cpu across sockets. Spread all pools on all sockets or try to be "packed" and spread only some pools ?